node.gyp

```diff
@@ -8,6 +8,7 @@
     'node_has_winsdk%': 'false',
     'node_shared_zlib%': 'false',
     'node_shared_http_parser%': 'false',
+    'node_shared_libcares%': 'false',
```

Comment:
The reason it's changed is that we now use the shared `configure_library` so I have to map it to what the `.pc` file contains. Should be fixable though.